docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
+- This package installs the [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) dependency for serializing/deserializing Json message payloads.
+- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart.
+:::
+
+![Add the Regression Games core SDK to your Unity project](img/installing-regression-games/installing-sdk.gif)
+
+## Add Extension Packages to Your Project
+
+We provide the following extensions for our core SDK to support common project dependencies.
+Each of these can be added to your project using a git url, similar to core package installation.
+
+### Animation Rigging Extension
+
+Adds support for Unity Animation Rigging.
+
+```
+https://github.com/Regression-Games/RGUnityBots?path=src/gg.regression.unity.bots.animation.rigging#v0.0.27
+```
+
+### Entity Component System (ECS) Extension
+
+Enables game state collection of Unity ECS Entities and their Components.
+Without this extension, the core SDK can still capture GameObject data in projects using ECS but will have no access to ECS Entities.
+Depending on your project and its needs, this extension may not be necessary, but we highly recommend including it
+for full visibility of your game's internal state.
+
+Capturing Entities and their Component data may negatively impact runtime performance.
+This capture can be toggled on and off through the Project Settings as needed.
+
+```
+https://github.com/Regression-Games/RGUnityBots?path=src/gg.regression.unity.bots.ecs#v0.0.27
+```
+
+## Authenticate through the Regression Games SDK
+
+Once our core SDK and applicable extensions have been added to your project, close the Package Manager.
+You will be greeted by a prompt for Regression Games login details.
+
+Create a personal account through [the Regression Games website](https://play.regression.gg/signup).
+This step is simple, and will give you access to many
+features such as saving [Gameplay Sessions](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and creating powerful [Validation Suites](../core-concepts/validation-suites/validation-suites-getting-started).
+
+:::caution
+You can submit your new account credentials through the Unity prompt now,
+but this method is less secure than creating and using an API key.
+We recommend taking an extra moment to follow the section below.
+:::
+
+### Set an API Key
+
+Generate a new API key through your account settings in the Regression Games client, and copy the generated value to your clipboard.
+
+![Generate a new API key](../core-concepts/img/api-keys/create-new-api-key.gif)
+
+:::info
+To learn more about our API keys, see the [Authenticating with API Keys](../core-concepts/authenticating-with-api-keys) section.
+:::
+
+Do not submit any information to the welcome prompt in Unity.
+Instead, close the prompt and navigate to `Edit` > `Project Settings` > `Regression Games`.
```

Comment:
Can we remove this? Or rather, this should be updated to reflect the changes in the new PR

Reply:
Thanks!